From: Jacob Martin <jacob.martin@canonical.com>
Date: Mon, 1 Jun 2026 16:36:01 +0800
Subject: In some environments, stdbuf is unable to load its shared library

and prints an innocuous error. The test is able to function even if stdbuf does
not, so ignore the message when it appears.
---
 run_test.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/run_test.sh b/run_test.sh
index a6c00f3..d96d5d9 100755
--- a/run_test.sh
+++ b/run_test.sh
@@ -225,6 +225,7 @@ generalize_expected_output() {
     sed -i "/^ERROR (dkms apport): /d" "${output_log}"
     # Swap any CC/LD/... flags (if set) with a placeholder message
     sed -i "s|\(make -j1 KERNELRELEASE=${KERNEL_VER} all\).*|\1 <omitting possibly set CC/LD/... flags>|" "${output_log}"
+    sed -i "\|^ERROR: ld.so: object '/usr/libexec/rust-coreutils/libstdbuf.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.$|d" "${output_log}"
 }
 
 run_with_expected_output() {
